modules/ROOT/pages/runtime-manager-api.adoc

```diff
@@ -1,15 +1,52 @@
-= About the Runtime Manager REST API
+= Runtime Manager API
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :keywords: cloudhub, cloudhub api, manage, api, rest
 
-The Runtime Manager REST API enables you to programmatically access much of the functionality provided by Runtime Manager. To deploy Mule applications using the Runtime Manager REST API, see https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/arm-rest-services[REST APIs].
+The Runtime Manager REST API enables you to programmatically access these functions of Runtime Manager:
```

Comment:
Are you sure you want to remove "REST" from the topic title? Or is there no other type of RM API?

Reply:
Added back - it wasn't in the TOC, but it is on Exchange.